app/src/main/res/layout/fragment_nutrition_product.xml

```diff
@@ -36,7 +36,15 @@
                 android:scaleType="fitCenter"
                 app:srcCompat="@drawable/ic_add_a_photo_black_48dp" />
 
-
+            <Button
+                android:drawableLeft="@drawable/ic_add_a_photo_black_18dp"
```

Comment:
Could you use the `BorderButton` style to ensure that all buttons have the same design? For example, see button style (change_ing_img) in fragment_ingredients_product.xml:

![image](https://user-images.githubusercontent.com/2354602/55683222-4eddcf00-58f2-11e9-8829-6a0e52b8a24d.png)